### PR TITLE
Add Joint Travel Regulations?

### PR DESCRIPTION
**Pull request by DrewStephensCoding**
Wanted to see if it would be possible to add the JTR as a subset to the pubs, the same way you did the AFROTC and Honor Guard pubs.  The JTR has been one of the most frequently requested publications, even though it's not strictly an AFI or Air Force pub.  I think it still has a good benefit being in the app....thoughts?

### CODE REVIEW

File: data.txt [hunk 1]
```diff
@@ -1 +1,7 @@
...
+[{"PubID":0,
```
Comment (willswire) [inline data.txt:1]: Hey @DrewStephensCoding! Thanks for the PR. Please reformat `data.txt` so that it is a single line of JSON formatted code.

**Review by willswire — CHANGES_REQUESTED** [review]
Please see the comment for `data.txt`

File: data.txt (hunk 1 above)
Comment (DrewStephensCoding) [reply data.txt:1]: Ah shoot, I used a site to prettify the json so I can read it 😂 had a hunch that might cause a problem! I'll fix it ASAP

**Comment by DrewStephensCoding** [comment]
@willswire I think I corrected it?

**Comment by willswire** [comment]
Looks good!

**Pull request merged**